### PR TITLE
feat: add weather service, and info in menu

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#371

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -108,6 +108,7 @@
     <script src="scripts/services/profile.js"></script>
     <script src="scripts/services/utils.js"></script>
     <script src="scripts/services/timedate.js"></script>
+    <script src="scripts/services/weather.js"></script>
```
Comment (ceoaliongroo) [inline client/app/index.html]: Install via bower

File: client/app/views/dashboard/main.menu.html [hunk 2]
```diff
@@ -16,6 +16,9 @@ <h3 class="menu-title">NegaWatt</h3>
 <span class="menu-timedate">{{timedate.today | amDateFormat: 'YYYY MMMM DD'}}</span>
 <span class="menu-separator"></span>
 <span class="menu-timedate">{{timedate.now | amDateFormat: 'H:mm'}}</span>
+<span class="menu-separator"></span>
+<span class="menu-weather" dir="ltr">{{weather.temp}} &deg;</span>
+<span class="menu-weather"><image src="{{weather.icon}}" alt="{{weather.description}}" title="{{weather.description}}"></image></span>
```
Comment (ceoaliongroo) [inline client/app/views/dashboard/main.menu.html]: src => ng-src

**Comment by ceoaliongroo** [comment]
@shahardo, please also create a test

**Comment by ceoaliongroo** [comment]
screen shot 2015-03-23 at 4 44 28 pm

File: client/app/scripts/directives/nw-weather/nw-weather.js [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    return {
+      restrict: 'EA',
+      templateUrl: 'scripts/directives/nw-weather/default.html',
+      controller: function(nwWeather, nwWeatherIcons) {
+        var weather = this;
+
+        // Request the weather data.
+        nwWeather.get(this.account).then(function(response) {
+          weather.data = response;
+          console.log(response.icon, nwWeatherIcons[response.icon]);
```
Comment (amitaibu) [inline client/app/scripts/directives/nw-weather/nw-weather.js]: remove

File: client/app/scripts/directives/nw-weather/nw-weather.js [hunk 4]
```diff
@@ -0,0 +1,316 @@
+'use strict';
+
+angular.module('angular-nw-weather', [])
+  .constant('openweatherEndpoint', 'http://api.openweathermap.org/data/2.5/weather')
+  .directive('nwWeatherTemperature', function () {
+    return {
+      restrict: 'EA',
+      templateUrl: 'scripts/directives/nw-weather/default.html',
```
Comment (amitaibu) [inline client/app/scripts/directives/nw-weather/nw-weather.js]: => template

**Comment by amitaibu** [comment]
merged.

**Pull request closed without merging**